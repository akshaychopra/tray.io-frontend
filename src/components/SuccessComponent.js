import DoneIcon from "@material-ui/icons/Done";
import { green } from "@material-ui/core/colors";
import { makeStyles } from "@material-ui/core/styles";

const useStyles = makeStyles((theme) => ({
  root: {
    color: theme.palette.success.main,
  },
  doneIcon: {
    display: "flex",
    fontSize: 80,
    color: green,
    width: "100%",
  },
}));

const SuccessComponent = () => {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <DoneIcon className={classes.doneIcon} />
      Please verify your email address, you should have received an email from us already
    </div>
  );
};
export default SuccessComponent;
