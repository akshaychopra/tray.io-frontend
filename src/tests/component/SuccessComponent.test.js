import React from "react";
import { render } from "@testing-library/react";
import SuccessComponent from "../../components/SuccessComponent";
import "@testing-library/jest-dom/extend-expect";
import renderer from "react-test-renderer";

test("Renders a success component", () => {
  const { getByText } = render(<SuccessComponent />);
  const linkElement = getByText(/Please verify your/i);
  expect(linkElement).toBeInTheDocument();
});

test("Renders registration form", () => {
  const tree = renderer.create(<SuccessComponent />).toJSON();
  expect(tree).toMatchSnapshot();
});
