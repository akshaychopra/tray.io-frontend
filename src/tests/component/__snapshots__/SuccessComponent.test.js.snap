// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders registration form 1`] = `
<div
  className="makeStyles-root-3"
>
  <svg
    aria-hidden={true}
    className="MuiSvgIcon-root makeStyles-doneIcon-4"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <path
      d="M9 16.2L4.8 12l-1.4 1.4L9 19 21 7l-1.4-1.4L9 16.2z"
    />
  </svg>
  Please verify your email address, you should have received an email from us already
</div>
`;
